addons/account/models/account_move.py

```diff
@@ -1017,29 +1017,31 @@ def compute_writeoff_counterpart_vals(values):
 
                 writeoff_lines.append(compute_writeoff_counterpart_vals(vals))
 
-            # Create balance line
-            writeoff_lines.append({
-                'name': _('Write-Off'),
-                'debit': total > 0 and total or 0.0,
-                'credit': total < 0 and -total or 0.0,
-                'amount_currency': total_currency,
-                'currency_id': total_currency and writeoff_currency.id or False,
-                'journal_id': journal_id,
-                'account_id': self[0].account_id.id,
-                'partner_id': partner_id
+            with self.env.norecompute():
```

Comment:
If the goal of the commit is to return only the writeoff line, why not do:
`line_to_reconcile += writeoff_move.line_ids.sorted(key='id')[:-1]`

If the writeoff move line is last from the list, it should have the largest ID.

Reply:
Indeed, it's a more minimal change.
Thanks